websites/R/Reverso/presence.ts

```diff
@@ -0,0 +1,72 @@
+      presenceData.state = translationText ? `Translating "${translationText}"` : 'Using translator';
+    }
+  }
+  if (domain === 'dl.reverso.net') {
+    presenceData.details = 'Download Reverso';
+    presenceData.state = '';
+  }
+  if (domain === 'www.corporate-translation.reverso.com') {
+    presenceData.details = 'Reverso Corporate';
+    if (pagename === '') presenceData.state = 'Homepage';
+  }
+  if ([].includes(domain)) {
+    if (pagename === '') presenceData.state = 'Homepage';
+  }
+  if (domain === 'synonyms.reverso.net') {
+    presenceData.details = 'Synonyms';
+    if (pagename === 'synonym') presenceData.state = 'Homepage';
+    if (pagename.startsWith('synonym/')) {
+      presenceData.details = `Synonyms (${(<HTMLSpanElement>document.querySelector('#trg-selector > span.option.front > span.lang-trg'))?.innerText || 'English'})`;
+      presenceData.state = `Searching "${(<HTMLInputElement>document.getElementById('searchbox'))?.value}"`;
+    }
+  }
+  if (["tasrif.reverso.net", "konjugator.reverso.net", "conjugator.reverso.net", "conjugador.reverso.net", "conjugueur.reverso.net", "pealim.reverso.net", "coniugazione.reverso.net", "doushi.reverso.net", "conjugacao.reverso.net", "glagol.reverso.net", "tasrif.reverso.net", "konjugator.reverso.net", "conjugator.reverso.net", "conjugador.reverso.net", "conjugueur.reverso.net", "pealim.reverso.net", "coniugazione.reverso.net", "doushi.reverso.net", "conjugacao.reverso.net", "glagol.reverso.net"].includes(domain)) {
+    presenceData.details = 'Conjugator';
+    presenceData.state = document.querySelector('.content-head') ? (<HTMLElement>document.querySelector('.content-head')).innerText : document.querySelector('title').innerText.split('|')[0];
+    if (['conjugation-english.html', '活用-英語.html', 'coniugazione-inglese.html', 'conjugacao-ingles.html'].includes(pagename)) presenceData.state = 'Homepage';
+    if (pagename.match(/conjugation-.+-verb-.+\.html/g)) {
+      presenceData.details = `Conjugator (${(<HTMLLabelElement>document.querySelector('#content-menu > div.tableForm.clearfix > div > div.select-wrap > a > label'))?.innerText || 'English'})`;
+      presenceData.state = `Searching "${(<HTMLInputElement>document.getElementById('txtVerb')).value}"`;
+    }
+    if (pagename === 'info-british-american-conjugation.html') {
+      presenceData.details = 'Reading a page:';
+      presenceData.state = 'British vs American English (verbs)';
+    }
+    if (pagename.match(/index-.+-\d+-\d+.html/g)) {
+      const pageTitleEl = <HTMLElement>document.querySelector('#form1 > div.content-conj.content-conj-inner > div > div.wrapperW > div.headerDescription.conj.clearfix > h1');
+      presenceData.details = pageTitleEl.innerText.split('from')[0] || 'Most common verbs';
+      presenceData.state = 'from ' + pageTitleEl.innerText.split('from')[1] || '';
+    }
+  }
+  presence.setActivity(presenceData, true);
+  presence.setTrayTitle();
```

Comment:
You set tray tiltle to nothing when you set activity to nothing as well, therefore remove it, and remove second param of setActivity() since you are not playing anything